test/e2e/data/shared/capv/main/metadata.yaml

```diff
@@ -6,13 +6,13 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
-  - major: 1
-    minor: 8
-    contract: v1beta1
   - major: 1
```

Comment:
I know we did this differently before. Can we make sure this file is identical to the top-level one? (I think it makes sense to test the same as users will use later)

Reply:
Similarly let's add all the old versions to the other metadata.yaml's around here